components/fatality-counter/component.js

```diff
@@ -9,6 +9,13 @@ const mq = facepaint(breakpoints.map(bp => `@media (min-width: ${bp}px)`));
 const ScrapdFatalityCounter = props => {
   const fatalityCount = props.fatalities.length;
 
+  let casesArray = new Array() 
+  for (var i = 0; i < props.fatalities.length; i++) {
+      casesArray.push(props.fatalities[i].case);
+  };
+    
+  let caseCount = new Set(casesArray).size;
```

Comment:
```js
let caseCount = new Set(casesArray).size; 
```
Should be:
```js
   let caseCount = new Set(props.fatalities).size; 
```

Reply:
`caseCount` should be `crashCount` to stay consistent with the terminology.

Reply:
Completed.